admin/app/components/stages/update-stage.js

```diff
@@ -46,6 +47,7 @@ export default class UpdateStage extends Component {
 
   async _updateStage() {
     const model = this.args.model;
+    model.message = this.form.message ? this.form.message.trim() : null;
```

Comment:
On aurait pu faire ça :

```suggestion
    model.message = this.form.message?.trim() ?? null;
```